package.json

```diff
@@ -13,7 +17,8 @@
       "webextensions"
     ],
     "ignores": [
-      "extension/vendor/**"
+      "extension/vendor/**",
+      "Gruntfile.js"
```

Comment:
You can use `// eslint-disable` before the snake case and `// eslint-enable` after.